msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -220,11 +220,12 @@ To view your app in Outlook on the web:
 
 To view your app running in Outlook for Android app:
 
-1. Open the Outlook app on your Android device and sign in using your developer tenant account. If the Outlook for Android app was already running prior to sideloading your app, restart Outlook to see it in the installed apps section.
-1. Select the **More** icon. Your sideloaded app appears among installed apps.
+1. Open the Outlook app on your Android device and sign in using your developer tenant account. If the Outlook app for Android was already running prior to sideloading your app, restart the Outlook app to see it in the installed apps section.
+
+1. Select the **Apps** icon. Your sideloaded app appears among installed apps.
 1. Select your app icon to open your app in Outlook for Android.
 
-    :::image type="content" source="images/outlook-mobile-more.png" alt-text="Screenshot shows the More option on the side bar of the Outlook app to see your installed personal tabs in Android.":::
+    :::image type="content" source="images/outlook-mobile-android-more.png" alt-text="Screenshot shows the App option on the side bar of the Outlook app to see your installed personal tabs in Android.":::
```

Comment:
- The **Apps** option
- I don't think it's called a "side bar" on mobile (and it's a horizontal "bar" along the bottom). How about "Screenshot shows the Apps option of the Outlook app on Android.")